shell/platform/linux/fl_platform_channel.h

```diff
@@ -0,0 +1,112 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef FLUTTER_SHELL_PLATFORM_LINUX_FL_PLATFORM_CHANNEL_H_
+#define FLUTTER_SHELL_PLATFORM_LINUX_FL_PLATFORM_CHANNEL_H_
+
+#include "flutter/shell/platform/linux/public/flutter_linux/fl_binary_messenger.h"
+#include "flutter/shell/platform/linux/public/flutter_linux/fl_method_call.h"
+
+G_BEGIN_DECLS
+
+typedef enum {
+  // NOLINTBEGIN(readability-identifier-naming)
+  FL_PLATFORM_CHANNEL_EXIT_TYPE_CANCELABLE,
+  FL_PLATFORM_CHANNEL_EXIT_TYPE_REQUIRED,
+  // NOLINTEND(readability-identifier-naming)
```

Comment:
Do we need to change enum naming lint rules for `//flutter/shell/platform/linux/*`? The linux embedder has its own .clang-tidy file here:
https://github.com/flutter/engine/blob/main/shell/platform/linux/.clang-tidy